integrationTests/multiShard/block/executingRewardMiniblocks_test.go

```diff
@@ -297,7 +303,37 @@ func updateNumberTransactionsProposed(
 	transactionsForLeader[addressProposer] += nbTransactions
 }
 
-func verifyRewards(
+func updateRewardsForMetachain(rewardsMap map[string]uint32, consensusNode *integrationTests.TestProcessorNode) {
+	metaRewardDataSlice := consensusNode.SpecialAddressHandler.ConsensusMetaRewardData()
+	if len(metaRewardDataSlice) > 0 {
+		for _, metaRewardData := range metaRewardDataSlice {
+			for _, addr := range metaRewardData.Addresses {
+				rewardsMap[addr]++
+			}
+		}
+	}
+}
+
+func verifyRewardsForMetachain(
+	t *testing.T,
+	mapRewardsForMeta map[string]uint32,
+	nodes map[uint32][]*integrationTests.TestProcessorNode,
+) {
+
+	// TODO this should be read from protocol config
+	rewardValue := uint32(1000)
```

Comment:
expose a function for test where you can change the reward value, than you can set anything.

Reply:
done